### ISSUE TITLE
Fix contrast for notes and unclear

### ISSUE THREAD

**Issue by nichgray** [CDRH/earlywashingtondc#249]
Notes and unclear are `#888`, which is creating a contrast error (see e.g. https://earlywashingtondc.org/doc/oscys.mdcase.0006.023 for contrast and https://earlywashingtondc.org/people/per.002074 for a source note). `gap` on the latter page and possibly other elements also fail contrast (in that case, with `#aaa`)
